README.md

```diff
@@ -1,2 +1,135 @@
+- `pip install awscli`
+- `conda install -c conda-forge awscli`
+- `brew install awscli`
+
+#### Configuring your AWS credentials
+
+When you set up your account, you may have gotten a paired `AWS Access Key ID` and `AWS Secret Access Key`.
+If you did not, you can login to the [aws console](https://console.aws.amazon.com/), select "My Security Credentials" from the menu that appears clicking on your username in the upper right.
+Scroll down to "Access keys for CLI, SDK, & API access" and click the "Create Access Key" button.
+You will see a popup with your Access key ID and a link to show the Secret access key.
+
+In the terminal, run the command `aws configure` and when prompted paste in the `AWS Access Key ID` and `AWS Secret Access Key`.
+You may want to set up your `Default region name`: most of the computing resources used here run in `us-east-1`, but setting this to a different region should not affect most commands.
+The `Default output format` can be left as `None`
+
+The `aws configure` command will create a directory at `~/.aws` with a `config` file and a `credentials` file that will be required for `nextflow` commands to work smoothly with Batch and S3.
+
+#### AWS infrastructure
+
+The infrastructure on AWS (Batch queues, AMIs, etc.) is defined via [terraform](https://www.terraform.io) using the files in the `aws` directory. More details are described in [aws/setup-log.md](aws/setup-log.md)
+
+
+
+## Running Workflows
+
+The workflows for this repository are stored in the `workflows` directory, organized by task into subdirectories.
+In general, workflows should be run from the main `workflows` directory, rather than the subdirectories.
+This will allow easy access to the shared `nextflow.config` file and keep all intermediate and log files in a single location.
+In particular, `nextflow` creates a separate work directory for every task: these will appear by default (for local tasks) in the `workflows/work` directory if the command is invoked from `workflows`.
+As the work directories can get large, it is helpful to have a single location to keep track of and purge as needed.
+
+Final output file locations are determined on a per-workflow basis, usually by the `params.outdir` setting within the script (possibly overridden by the `--outdir` option).
+In most cases this will be an S3 bucket within `s3://nextflow-ccdl-results`.
+
+## Running locally
+
+⚠️⚠️⚠️
+Running workflows in this repository locally is not something to take lightly!
+It is fine for a quick test, and useful for development, but note that most of the workflows here use very large data files, which will have to be downloaded locally to run.
+Workflow processing may require large amounts of RAM and time, so the following example commands will rarely be used, and are mostly for illustration.
+⚠️⚠️⚠️
+
+The basic command to run a workflow will look something like the following:
+
+```
+nextflow run checks/check-md5.nf
```

Comment:
I like this example as opposed to the old one. It seems to have run successfully. It took my computer 1h 7m. If that's a normal run time, may want to put a note that this does take awhile to run at first.